wfl-slack-app/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
It feels like we should be excluding .vscode from git.  These are local specific files.  Everything could probably be re-used by someone else if they chose to use vscode too though...